pkg/imgpkg/registry/registry.go

```diff
@@ -122,6 +122,13 @@ func NewSimpleRegistry(opts Opts, regOpts ...regremote.Option) (*SimpleRegistry,
 		regRemoteOptions = append(regRemoteOptions, regOpts...)
 	}
 
+	regRemoteOptions = append(regRemoteOptions, regremote.WithRetryBackoff(regremote.Backoff{
+		Duration: 1.0 * time.Second,
+		Factor:   3.0,
+		Jitter:   0.1,
+		Steps:    opts.RetryCount,
+	}))
```

Comment:
i think that's where im leaning towards is we should just keep retry backoff more linear, retry after [100ms, 300ms, 500ms, 1s (cap at 1s)]